traptor/traptor.py

```diff
@@ -593,11 +608,12 @@ def _main_loop(self):
 
                     if self.kafka_enabled == 'true':
                         try:
-                            self.kafka_conn.send(self.kafka_topic, enriched_data)
-                            self.logger.info("Tweet sent to kafka: {}".format(tweet.get('id_str', None)))
-                            dd_monitoring.increment('tweet_to_kafka_success')
+                            self.logger.info("Attempting to send tweet {} to kafka".format(tweet.get('id_str', None)))
+                            future = self.kafka_conn.send(self.kafka_topic, enriched_data)
+                            future.add_callback(self.kafka_success_callback, tweet)
```

Comment:
The tweet in the success callback is only used to yet the id. we can pass in the id directly instead if you'd like.